pkg/publish/utils.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package publish
+
+import (
+	"fmt"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/ko/pkg/build"
+	"os"
+)
+
+//ToImage builds the image reference from the build result
+func ToImage(br build.Result, s string) (v1.Image, error) {
```

Comment:
```suggestion
func toImage(br build.Result, s string) (v1.Image, error) {
```

(In general, I'd prefer not to export anything we don't want to be usable outside this package)